src/common/mfu_daos.c

```diff
@@ -2797,8 +2798,6 @@ static int serialize_recx_array(struct hdf5_args *hdf5,
             /* created attribute in HDF5 file with encoded
              * dataspace for this record extent */
             memset(attr_name, 0, ATTR_NAME_LEN);
-            memset(number_str, 0, ATTR_NAME_LEN);
-            memset(attr_num_str, 0, ATTR_NAME_LEN);
```

Comment:
It looks like attr_name is also statically set to 0 on declaration as well, so could we remove this?
Or is it because we're in a loop?

Reply:
@daltonbohning I left it because it is a loop, although I'm not sure if it is needed.

Reply:
@daltonbohning I removed it